### PR TITLE
Highlights from Inclusive Design 24

### PR DESCRIPTION
**Pull request by hxmattobee**
### Blog title
Highlights from Inclusive Design 24 (2017)

### Blog description
Inclusive Design 24 is a free online accessibility conference, organised annually by The Paciello Group. The theme this year was about going beyond the basics and beyond compliance, and featured presentations from people at Facebook, Microsoft the BBC. I’ve finally found the time to write up my notes.

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2017-08-14-inclusive-design-24.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Highlights from Inclusive Design 24 (2017)
+categories: accessibility
+author: matt_obee
+tags: accessibility
+comments: true
+excerpt: "With 24 one-hour webinars from a range of speakers, ID24 is always packed with insights and ideas for beginners and experts alike. The theme this year was about going beyond the basics and beyond compliance, and featured presentations from people at Facebook, Microsoft the BBC."
```
Comment (jackdcrawford) [inline _posts/2017-08-14-inclusive-design-24.md]: Microsoft **and** the BBC

File: _posts/2017-08-14-inclusive-design-24.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+[Inclusive Design 24](http://www.inclusivedesign24.org/) is a free online accessibility conference, organised annually by The Paciello Group. With 24 one-hour webinars from a range of speakers, ID24 is always packed with insights and ideas for beginners and experts alike. The theme this year was about going beyond the basics and beyond compliance, and featured presentations from people at Facebook, Microsoft the BBC. I've finally found the time to write up my notes, so what did I learn from this year's speakers?
```
Comment (jackdcrawford) [inline _posts/2017-08-14-inclusive-design-24.md]: Microsoft **and** the BBC

File: _posts/2017-08-14-inclusive-design-24.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Government going beyond compliance
+[Jason Kiss from New Zealand](https://www.youtube.com/watch?v=xFbX4QgPFvs) and [Andrew Arch from Australia](https://www.youtube.com/watch?v=A2emdjBHkyY) both talked about efforts to move beyond technical conformance and compliance in their respective government teams.
+
+New Zealand government guidelines were originally based on WCAG 1.0, before being updated to reflect WCAG 2.0. Government agencies were asked to self assessed themselves in 2011 and 2014, but the results were difficult to to interpret. Both rounds of audits required a considerable investment of time and money. Moving forward, instead of treating accessibility as a compliance issue, the aim was to raise accessibility awareness, knowledge and skill across agency teams. They moved away from complex formal audits to free tools and a limited range of manual checks that required minimal technical knowledge. The tests were grouped by topic instead of WCAG success criteria and they also avoided absolute pass/fail tests, with a greater emphasis on how those issues might affect the user.
```
Comment (mattsaunby) [inline _posts/2017-08-14-inclusive-design-24.md]: In the second sentence here, the word 'assessed' should only be 'assess' and later in the same sentence the word 'to' is repeated.

**Comment by mattsaunby** [comment]
Changes look good mate :) 👍

**Pull request merged**